### ISSUE TITLE
[FLUX TRAIN] is there option for val loss?

### ISSUE THREAD

**Issue by SlZeroth** [ostris/ai-toolkit#57]
SimpleTuner support the 

max_grad_norm=1.0
VALIDATION_RESOLUTION , VALIDATION_GUIDANCE  , VALIDATION_GUIDANCE_REAL 

for val loss

I cant found option in ostris/ai-toolkit 

I trained person but I got overfitting result.

If most of the portrait photos in the training data include hands, when generating an image with the prompt 'portrait of ohwx woman', it will generate an image that includes hands.
Simpletuner doesn't have this problem. What kind of issue is this??

**Comment by WarAnakin** [comment]
when it comes to flux training simple tuner is as good as that thing you flush down your toilet (trust me i have THOUSANDS of hours of training accumulated)
what they mean by validation is the same as saying `sample image cfg` `sample image resolution` 
The terminology is irrelevant. In ai-toolkit validation you can consider validation everything below the `sample` sub-category

![image](https://github.com/user-attachments/assets/2a6056ab-131f-47d0-8f9f-963817b89a84)
